<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CorrelationBayesianForm</class>
 <widget class="QWidget" name="CorrelationBayesianForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_12">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="widget_4" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="1" column="1">
       <widget class="AssignButton" name="assignButton">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="3">
       <widget class="BoundListView" name="variables">
        <property name="maximumSize">
         <size>
          <width>210</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="3">
       <widget class="AvailableFieldsListView" name="availableVariables">
        <property name="maximumSize">
         <size>
          <width>210</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QWidget" name="widget_5" native="true">
     <layout class="QGridLayout" name="gridLayout_11">
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="BoundGroupBox" name="bayesFactorType" native="true">
        <property name="title" stdset="0">
         <string>Bayes factor</string>
        </property>
        <property name="flat" stdset="0">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButton">
           <property name="text">
            <string>BF₁₀</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="radioButton_2">
           <property name="text">
            <string>BF₀₁</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="BoundGroupBox" name="hypothesis" native="true">
        <property name="title" stdset="0">
         <string>Hypothesis</string>
        </property>
        <property name="flat" stdset="0">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QRadioButton" name="_1correlated">
           <property name="text">
            <string>Correlated</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="_3correlatedNegatively">
           <property name="text">
            <string>Correlated negatively</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="_2correlatedPositively">
           <property name="text">
            <string>Correlated positively</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QWidget" name="widget_7" native="true">
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <item row="4" column="0">
       <widget class="QWidget" name="credibleIntervalsIntervalContainer" native="true">
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QWidget" name="widget_8" native="true">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="BoundTextBox" name="credibleIntervalsInterval">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Interval</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>%</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="BoundCheckBox" name="flagSupported">
        <property name="text">
         <string>Flag supported correlations</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="BoundCheckBox" name="reportBayesFactors">
        <property name="text">
         <string>Report Bayes factors</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="BoundCheckBox" name="credibleIntervals">
        <property name="text">
         <string>Credible Intervals</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="groupPrior" native="true">
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="1">
       <widget class="BoundTextBox" name="priorWidth">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Beta* prior width</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QWidget" name="optionsWidget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="ExpanderButton" name="buttonOptions">
        <property name="text">
         <string>Options</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="panelOptions" native="true">
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="1" column="0">
          <widget class="BoundGroupBox" name="missingValues" native="true">
           <property name="title" stdset="0">
            <string>Missing values</string>
           </property>
           <property name="flat" stdset="0">
            <bool>true</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_9">
            <property name="topMargin">
             <number>6</number>
            </property>
            <property name="bottomMargin">
             <number>6</number>
            </property>
            <item row="0" column="0">
             <widget class="QRadioButton" name="_1excludePairwise">
              <property name="text">
               <string>Exclude cases pairwise</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="_2excludeListwise">
              <property name="text">
               <string>Exclude cases listwise</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1" rowspan="3">
          <widget class="QWidget" name="widget_6" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="GroupBox" name="groupBox_3">
           <property name="title">
            <string>Statistics</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="BoundCheckBox" name="meansAndStdDev">
              <property name="text">
               <string>Means and standard deviations</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="BoundCheckBox" name="crossProducts">
              <property name="text">
               <string>Cross-product deviations and covariances</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AssignButton</class>
   <extends>QPushButton</extends>
   <header>widgets/assignbutton.h</header>
  </customwidget>
  <customwidget>
   <class>AvailableFieldsListView</class>
   <extends>QListView</extends>
   <header>widgets/availablefieldslistview.h</header>
  </customwidget>
  <customwidget>
   <class>BoundListView</class>
   <extends>QListView</extends>
   <header>widgets/boundlistview.h</header>
  </customwidget>
  <customwidget>
   <class>ExpanderButton</class>
   <extends>QPushButton</extends>
   <header>widgets/expanderbutton.h</header>
  </customwidget>
  <customwidget>
   <class>BoundCheckBox</class>
   <extends>QCheckBox</extends>
   <header>widgets/boundcheckbox.h</header>
  </customwidget>
  <customwidget>
   <class>GroupBox</class>
   <extends>QGroupBox</extends>
   <header>widgets/groupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BoundTextBox</class>
   <extends>QLineEdit</extends>
   <header>widgets/boundtextbox.h</header>
  </customwidget>
  <customwidget>
   <class>BoundGroupBox</class>
   <extends>QWidget</extends>
   <header>widgets/boundgroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
